atomicapp/providers/openshift.py

```diff
@@ -332,10 +333,10 @@ class OpenShiftProvider(Provider):
     provider_ca = None
 
     # Parsed artifacts. Key is kind of artifacts. Value is list of artifacts.
-    openshift_artifacts = {}
+    openshift_artifacts = defaultdict(list)
 
     def init(self):
-        self.openshift_artifacts = {}
+        self.openshift_artifacts = defaultdict(list)
```

Comment:
https://github.com/projectatomic/atomicapp/commit/c5e6f27c76823343351a3c048f5efde13e86f44d is fine, this is needed, @dustymabe is also asking about https://github.com/projectatomic/atomicapp/pull/680/files#diff-b41fe9caf01c4b4c9724d8a1fd6325baR336